snipeit.sh

```diff
@@ -228,11 +221,14 @@ install_composer () {
 install_snipeit () {
   create_user
   echo "* Creating MariaDB Database/User."
-  mysql -u root --execute="CREATE DATABASE snipeit;GRANT ALL PRIVILEGES ON snipeit.* TO snipeit@localhost IDENTIFIED BY '$mysqluserpw';"
+  mysql -u root --execute="CREATE DATABASE snipeit;CREATE USER snipeit_dbuser@localhost IDENTIFIED BY '$mysqluserpw'; GRANT ALL PRIVILEGES ON snipeit.* TO snipeit_dbuser@localhost;"
 
   echo -e "\n\n* Cloning Snipe-IT from github to the web directory."
   log "git clone https://github.com/snipe/snipe-it $APP_PATH" & pid=$!
   progress
+  pushd $APP_PATH
+  git checkout master
```

Comment:
This should stay `master`, and then switch to `main`, right?

Reply:
That's how I was thinking about it. Wait till we've officially landed on main before we switch, otherwise it's broken until then, and Gosh forbid we need to delay the switch for some reason

Reply:
That's what I was thinking too, just making sure we're on the same page. :)